lib/beaker/hypervisor/google_compute_helper.rb

```diff
@@ -393,13 +395,29 @@ def create_instance(name, img, machine_type, disk_size)
         ),
       ],
     )
-    new_instance = ::Google::Apis::ComputeV1::Instance.new(
-      machine_type: machine_type.self_link,
-      name: name,
-      disks: [disk_params],
-      network_interfaces: [network_interface],
-      tags: tags,
-    )
+
+    # use custom hostname if specified
+    if hostname && ENV.fetch('BEAKER_set_gce_hostname', false)
+      # The google api requires an FQDN for the custom hostname
+      hostname.include? '.' ? valid_hostname = hostname : valid_hostname = hostname + '.beaker.test' 
+
+      new_instance = ::Google::Apis::ComputeV1::Instance.new(
```

Comment:
I thin it's nicer to define the parameters for `Google::Apis::ComputeV1::Instance` in the if/else block and define `Google::Apis::ComputeV1::Instance` afterwards once, with the correct parameters. We use the same pattern in Puppet code as well.